apps/frontend/src/components/global/ExportCKLModal.vue

```diff
@@ -585,14 +589,12 @@ export default class ExportCKLModal extends Vue {
   }
 
   validateFormat(field: typeof ValidationProperties, name: string): string[] {
-    const errors: string[] = [];
     const dirty = _.get(field, '$dirty');
     const invalid = _.get(field, '$invalid');
-    if (!dirty) {
+    if (!dirty || !invalid) {
```

Comment:
After discussion, we have no idea what dirty means.  Please research.

Reply:
The use of `dirty` is unclear and unnecessary. It will be removed. [https://vuelidate.js.org/#sub-v-values](https://vuelidate.js.org/#sub-v-values)

Reply:
Done.